src/content/css-geometry-tools.js

```diff
@@ -61,33 +55,27 @@ function addMargin(elem, rect, win) {
 /**
  * Returns a DOMRect for the content rect of the element, in float CSS pixels.
  * Returns an empty rect if the object has zero or more than one client rects.
- * @param  {object} elem The DOM element whose content rect to compute.
- * @return {object}      The element's content rect.
  */
-export function getContentRect(elem) {
-  const clientRects = elem.getClientRects();
+export function getContentRect(element: HTMLElement): DOMRectInterface {
+  const clientRects = element.getClientRects();
   if (clientRects.length !== 1) {
     return new DOMRect();
   }
 
   const borderRect = clientRects[0];
-  const win = elem.ownerDocument.defaultView;
-  return subtractPadding(elem, subtractBorder(elem, borderRect, win), win);
+  return subtractPadding(element, subtractBorder(element, borderRect));
 }
 
 /**
  * Returns a DOMRect for the margin rect of the element, in float CSS pixels.
  * Returns an empty rect if the object has zero or more than one client rects.
- * @param  {object} elem The DOM element whose margin rect to compute.
- * @return {object}      The element's margin rect.
  */
-export function getMarginRect(elem) {
-  const clientRects = elem.getClientRects();
+export function getMarginRect(element: HTMLElement): DOMRectInterface {
+  const clientRects = element.getClientRects();
   if (clientRects.length !== 1) {
     return new DOMRect();
   }
 
   const borderRect = clientRects[0];
-  const win = elem.ownerDocument.defaultView;
-  return addMargin(elem, borderRect, win);
+  return addMargin(element, borderRect);
```

Comment:
Were the changes in this file supposed to be in this PR or have you done this elsewhere and included them by mistake ?